downloader/src/main/kotlin/vcs/Git.kt

```diff
@@ -20,6 +20,7 @@
 package org.ossreviewtoolkit.downloader.vcs
 
 import com.jcraft.jsch.JSch
+import com.jcraft.jsch.ProxyHTTP
```

Comment:
Commit message:

> - Set proxy for git in case of reverse proxy 

Please omit the leading "- " and s/git/Git/

Reply:
Done